src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -1874,7 +1874,7 @@ def load_arguments(self, _):
 
     with self.argument_context('network traffic-manager profile check-dns') as c:
         c.argument('name', name_arg_type, help='DNS prefix to verify availability for.', required=True)
-        c.argument('type', ignore_type, default='Microsoft.Network/trafficManagerProfiles')
+        c.argument('resource_type', ignore_type, default='Microsoft.Network/trafficManagerProfiles')
```

Comment:
Why you rename it to `resource_type`? It will cause breaking change which is avoidable.